CHANGELOG.md

```diff
@@ -147,9 +147,9 @@ not rely on the `x/supply` keeper to get module account addresses for blacklisti
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmwasm/wasmd/compare/v2.0.3...HEAD
-[v2.0.3]: https://github.com/cosmwasm/wasmd/releases/tag/v2.0.3
-[v2.0.2]: https://github.com/cosmwasm/wasmd/releases/tag/v2.0.2
-[v2.0.1]: https://github.com/cosmwasm/wasmd/releases/tag/v2.0.1
-[v2.0.0]: https://github.com/cosmwasm/wasmd/releases/tag/v2.0.0
-[v1.0.0]: https://github.com/cosmwasm/wasmd/releases/tag/v1.0.0
+[Unreleased]: https://github.com/CosmWasm/wasmd/compare/v2.0.3...HEAD
```

Comment:
This CHANGELOG is odd as well cuz it refers to gaia numbers.

We can reset all this when wasmd hits 1.0, for now I wouldn't spend too much time on this.